DMTN-049.tex

```diff
@@ -666,4 +648,117 @@ \subsection{Considerations for Maximizing Early Science}\label{ssec:use_LOY1}
 
+\subsection{A Summary of Scientific Recommendations from the LORs}\label{ssec:use_scilor}
+
+As described in \S~\ref{ssec:lor_choice}, of the 6 LOR describing scientific use cases:
+\begin{itemize}
+\item 1 focused on galaxies
+\item 2 focused on dark energy
+\item 3 focused on active galactic nuclei
+\end{itemize}
+
+Recommendations that are within scope for the DM team (or were already in the plans or evaluation criteria; \S~\ref{sec:eval}) and are likely to be achieved:
+\begin{itemize}
+\item adopt a PZ estimator that provides PDFs, point estimates, SED/template fits, physical params, and flags
+\item adopt at least one PZ estimator that can generate accurate PZ for galaxies with AGN
+\item adopt a PZ estimator that performs well with imperfect priors / incomplete training sets (especially at low-z) 
+\item provide PZ in the Object catalog at the time of data release (i.e., no delay)
+\item make all PZ-related code, inputs, metadata, etc. available (for reproducibility)
+\item correct for MW dust before passing fluxes to the PZ estimator
+\item provide users with a framework and tools to cross-match to non-LSST catalogs
+\end{itemize}
+Any of these recommendations which were not already part of the evaluation criteria in \S~\ref{sec:eval} have been added.
+
+Recommendations that present a challenge:
+\begin{itemize}
+\item incorporate non-LSST, non-optical photometry into the PZ estimates
+\item cross-match the LSST Objects catalogs with external data sets (e.g., an X-ray AGN catalog) and provide the results with flags and identifiers
+\item release revised PZ data products every few months as new sky regions are observed
+\item link Object identifiers between data releases (so users can get new host data for old alerts)
+\item in the alert packets, provide host galaxy Object data (PZ, photometry, shape) and non-LSST survey data for host galaxies (spec-z, PZ, photometry)
+\end{itemize}
+These challenges are out of scope for the Rubin Construction project (and some violate the current Rubin data rights policy), and some items (such as providing cross-matched catalogs) are best left to the expertise of the science community.
+However, just because an idea or data product presents a challenge, or is beyond the scope of Rubin construction and commissioning, does not mean it could never be accomplished.
+Adopting and federating value-added user-generated data products, and updating algorithms in the LSST Science Pipelines, is an Operations-era (2024 and beyond) activity that will take input from the science community via the Rubin Users Committee, the Community Engagement Team, the Science Advisory Committee, and other such groups.
+It is very useful to have these suggestions raised early on so that the DM team can help the science community prepare for Operations, all of the LOR providing these recommendations are much appreciated.
```

Comment:
```suggestion
It is very useful to have these suggestions raised early on so that the DM team can help the science community prepare for Operations; all of the LOR providing these recommendations are much appreciated.
```